packages/core/plugins/core-plugins-server-internal/src/discovery/plugins_discovery.ts

```diff
@@ -64,10 +66,43 @@ export function discover({
       return typeof pluginPathOrError === 'string'
         ? createPlugin$(pluginPathOrError, log, coreContext, instanceInfo, nodeInfo)
         : [pluginPathOrError];
-    }),
-    shareReplay()
+    })
   );
 
+  const pluginPkgDiscovery$ = from(coreContext.env.repoPackages ?? EMPTY).pipe(
+    filter(
+      getPluginPackagesFilter({
+        oss: coreContext.env.cliArgs.oss,
+        examples: coreContext.env.cliArgs.runExamples,
+        paths: config.additionalPluginPaths,
+        parentDirs: config.pluginSearchPaths,
+      })
+    ),
+    map((pkg) => {
+      log.debug(`Successfully discovered plugin package "${pkg.id}"`);
+      const manifest = pluginManifestFromPluginPackage(
+        coreContext.env.packageInfo.version,
+        pkg.manifest
+      );
```

Comment:
It's still unclear to me how to define multi-zone (browser+server) plugins with this package approach? Is it as simple as creating a package with a `server` and `client` sub folders?

Reply:
Sorry if my question was unclear, let me rephrase: can we still define plugins having both `server` and `client` parts with the new system?